src/components/internal/InternalInput.tsx

```diff
@@ -58,6 +59,7 @@ const InternalInput = (props: InternalInputProps) => {
     parentName,
     innerRef,
     id,
+    testId,
```

Comment:
We shouldn't need `testId`s on internal ocmponents. Let's chat about this.

Reply:
see above comment